FordLabs/retroquest
# Fix CSV download

## Overview
When we swapped to using `List.of()` for generating the list of strings for the CSV, we missed that `List.of()` doesn't accept nulls and the assignee for an Action Item can be null. This just returns an empty string if there is no assignee and moves the logic for generating the CSV line per item out of the Thought and Action Item Entities.

## Testing Instructions
1. Add a new Action Item without an Assignee
2. Click the download CSV button
3. Verify the file contains the expected information for a Thought and an Action Item
